docs/deployment/add-ons/load-balancer/README.md

```diff
@@ -0,0 +1,111 @@
+
+## Creating the load balancer
+
+To make it easy to create the load balancer, you can use the [script](#run-the-script) provided in the instructions below. Following is a brief overview of what happens when you run the script:
+### Domain and Certificate creation
+
+As explained in the background section, you need a registered domain and TLS certificate to use HTTPS with load balancer. Since your top level domain(e.g. `example.com`) could have been registered at any service provider, for uniformity and taking advantage of the integration provided between Route53, ACM and Application Load Balancer, you will create a separate [sudomain](https://en.wikipedia.org/wiki/Subdomain) (e.g. `platform.example.com`) to host Kubeflow and a corresponding hosted zone in Route53 to route traffic for this subdomain. To get TLS support, you will need certificates for both the root domain(`*.example.com`) and subdomain(`*.platform.example.com`).
+
+### Ingress and Load Balancer Controller configuration
+
+After creating the subdomain and certificates, you will need to:
+- configure the parameters for [ingress](../../../../awsconfigs/common/istio-ingress/overlays/https/params.env) with the certificate ARN of the subdomain)
+- tag the cluster subnets as described in `Prerequisites` section in this [document](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html)
+- configure the parameters for [load balancer controller](../../../../awsconfigs/common/aws-alb-ingress-controller/base/params.env) with the cluster name 
+- create an IAM role with [these permissions](../../../../awsconfigs/infra_configs/iam_alb_ingress_policy.json) for the load balancer controller to use via a service account(`alb-ingress-controller` service account in `kubeflow` namespace). You can use the [eksctl create iamserviceaccount](https://eksctl.io/usage/iamserviceaccounts/) command for this.
+
+You can then use `kubectl` and `kustomize` to build and install the components specified in this [kustomize](./kustomization.yaml) file.
+
+After installing the components, the script will wait for the ingress managed load balancer to be created and update the subdomain (`platform.example.com`) with a CNAME entry for `kubeflow.platform.example.com` pointing to the load balancer DNS. You can see the load balancer DNS by running `kubectl get ingress -n istio-system istio-ingress`.
+
+To summarize, what the script does or what you need to if you are doing it manually:
+    - Create a custom domain to host Kubeflow
+    - Create TLS certificates for the domain using ACM
+    - Configure the ingress and load balancer controller
+    - Build manifests and install the required components
+    - Update the domain with ALB address
+
+### Run the script
+
+1. Install dependencies for the script
+    ```
+    cd tests/e2e
+    pip install -r requirements.txt
+    ```
+1. Substitute values in `tests/e2e/utils/load_balancer/config.yaml`.
```

Comment:
I am lost here, are we writing this steps for developer to test this feature ? or we writing this for end user ?

Reply:
answered above